File: PYTHON/student_grades.py
def student_grades():
    number_of_students = int(input("How many students do you have?: "))
    number_of_subjects = int(input("How many subjects do they offer?: "))

    scores = number_of_students + number_of_subjects
    total_scores = number_of_students
    average_scores = number_of_students
    positions = number_of_students

    for i in range(number_of_students):
        print(f"Enter scores for Student {i + 1}:")
        total = 0
        for j in range(number_of_subjects):
            score = int(input(f"Subject {j + 1}: ")) 
            total += scores
        
        total_scores = total
        average_scores = total / number_of_subjects

    print("\n==========================================================================")
    print(f"{'STUDENT':<10}", end="")

    for i in range(1, number_of_subjects + 1):
        print(f"{'SUB'+str(i):>10}", end="")

    print(f"{'TOT':>10} {'AVE':>10} {'POS':>10}")
    print("==========================================================================")

    for i in range(1, number_of_students + 1):
        print(f"Student {i:<10}")

    print(f"{total_scores:>10} {average_scores:>10.2f} {positions:>10}")

File: JAVASCRIPT/studentGrades.js
function studentGrades() {
    const numberOfStudents = Int(prompt("How many students do you have?: "));
    const numberOfSubjects = Int(prompt("How many subjects do they offer?: "));

    let scores = numberOfStudents + numberOfSubjects;
    let totalScores = numberOfStudents;
    let averageScores = numberOfStudents;
    let positions = numberOfStudents;

    for (let i = 0; i < numberOfStudents; i++) {
        console.log(Enter scores for Student {i + 1}:);
        let total = 0;
        for (let j = 0; j < numberOfSubjects; j++) {
            let score = Int(prompt(Subject {j + 1}: ));
            total += scores;
        }

        totalScores = total;
        averageScores = total / numberOfSubjects;
    }

    console.log("\n==========================================================================");
    console.log("STUDENT".(10), "");

    for (let i = 1; i <= numberOfSubjects; i++) {
        process.stdout.write("SUB" + i.(10));
    }

    console.log("TOT".(10), "AVE".(10), "POS".(10));
    console.log("==========================================================================");

    for (let i = 1; i <= numberOfStudents; i++) {
        console.log(Student {i});
    }

    console.log({totalScores(10)} {averageScores(10)} {positions(10)});
}

